app/Http/Controllers/Front/CheckoutController.php

```diff
@@ -228,9 +228,12 @@ public function postCheckout(Request $request)
                     $attributes = $this->getAttributes($content);
                 }
             } else {
+                
                 $items = new \Illuminate\Support\Collection();
                 $invoiceid = $request->input('invoice_id');
                 $invoice = $this->invoice->find($invoiceid);
+                $processingFee = $payment_method =="razorpay"? 0 : $request->input('process_fee');
```

Comment:
`$request->input('process_fee')` is this coming from client panel?

Reply:
Getting value now in backend